### PR TITLE
Installer: promote altinfra image

### PR DESCRIPTION
**Pull request by patrickdillon**
This alternate Installer image will be free of Terraform dependencies in order to mitigate any CVE issues.

Depends on openshift/installer#7656

### CODE REVIEW

**Comment by patrickdillon** [comment]
Was looking for guidance on how to add mappings and just discovered https://github.com/openshift/release/blob/388047f020e7dfff64ca3529c4bab9f50d4d2e19/core-services/image-mirroring/openshift/GUIDELINES.md so I will update this accordingly.

**Comment by bparees** [comment]
per conversation with @patrickdillon this image is temporary, so the name is fine.
/approve

**Comment by patrickdillon** [comment]
/hold

I dropped the Dockerfile from the installer repo so need to re-add the build tag here.

**Comment by patrickdillon** [comment]
/hold cancel
Updates have been made to reintroduce the build tag.

File: ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml [hunk 1]
```diff
@@ -72,26 +58,10 @@ resources:
...
-    requests:
-      cpu: "3"
-      memory: 5Gi
-  upi-installer:
-    requests:
-      cpu: "3"
-      memory: 5Gi
 test_binary_build_commands: |
   # Some agent-based installer tests require nmstatectl
   dnf install -y nmstate
```
Comment (andfasano) [inline ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml:67]: Without any agent related tests then this could be skipped

Comment (patrickdillon) [reply ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml]: thanks
done

**Comment by patrickdillon** [comment]
ug
/pj-rehearse ack

**Pull request merged**